bee-plugin/src/streamer.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::grpc::{plugin_client::PluginClient, DummyEvent, SillyEvent};
+
+use tokio::{
+    select,
+    sync::{mpsc::UnboundedReceiver, oneshot::Receiver},
+};
+
+use tokio_stream::wrappers::UnboundedReceiverStream;
+
+use tonic::transport::Channel;
+
+pub(crate) struct PluginStreamer<T> {
```

Comment:
I could be helpful to explain the `PluginStreamer` abstraction a bit in a few lines of doc comments. It's not obvious immediatedly from the type name (to me at least) what its exact job description is.